src/components-examples/cdk/listbox/cdk-listbox-disabled/cdk-listbox-disabled-example.html

```diff
@@ -1,7 +1,9 @@
 <form>
   <p>
-    <input type="checkbox" [formControl]="canDrinkCtrl">&nbsp;
-    <label>I am 21 or older</label>
+    <label>
```

Comment:
I think the correct fix would be to set a `for` attribute on the `label`.

Reply:
I saw both usages in the repo, but sure. I'm gonna chang it for `id="can-drink"` as in the `checkbox-demo.html`.